cassava_trait.obo

```diff
@@ -7594,6 +7594,139 @@ def: "Aroma like tha of sweet potato, Possibly like aroma of boiled, roasted or
 synonym: "BoiCasAromSweetpotat" EXACT []
 is_a: CO_334:0103020 ! Taste trait
 
+[Term]
+id: CO_334:0003040
+name: Fufu colour scale 1-8
+namespace: CassavaTrait
+def: "Measures the intensity of the different colours of the cooked fufu surface on a scale 1 to 8" []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003047 ! fufu surface colour
+relationship: variable_of CO_334:0013005 ! appearance method
+relationship: variable_of CO_334:0103022 ! Colour scale 1-8
+created_by: Ugo Chijioke
+
+[Term]
+id: CO_334:0003041
+name: Fufu smoothness scale 0-10
+namespace: CassavaTrait
+def: "Measures the absence of lumps when a sample is pressed firmly between the fingers" []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003048 ! fufu smoothness in hand
+relationship: variable_of CO_334:0013006 ! texture by handfeel method
+relationship: variable_of CO_334:0103023 ! Smoothness scale 0-10
+created_by: Ugo Chijioke
+
+[Term]
+id: CO_334:0003042
+name: Fufu stickiness scale 0-10
+namespace: CassavaTrait
+def: "Measures the adherence of fufu onto the fingers when it is touched with fingers or during moulding into bolus for swallowing" []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003049 ! fufu stickiness in hand
+relationship: variable_of CO_334:0013007 ! texture by handfeel method
+relationship: variable_of CO_334:0103024 ! Stickiness scale 0-10
+created_by: Ugo Chijioke
+
+[Term]
+id: CO_334:0003043
+name: Fufu moldability scale 0-10
+namespace: CassavaTrait
+def: "Meassures the easiness of fufu to stick together without disintegrating easiness to mould." []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003050 ! fufu moldability in hand
+relationship: variable_of CO_334:0013008 ! texture by handfeel method
+relationship: variable_of CO_334:0103025 ! Moldability scale 0-10
+created_by: Ugo Chijioke
+
+[Term]
+id: CO_334:0003044
```

Comment:
is this similar to this scale? 
https://github.com/Planteome/CO_334-cassava-traits/blob/3b05da7d5a44d0b02f8f74702c66329f5f4c78c4/cassava_trait.obo#L12027

Reply:
we will merge the IDs as we agree that they are the same: a 0-10 scale for hardness, even if the products are different